Gems/Atom/RHI/Vulkan/Code/Source/RHI/CommandQueue.cpp

```diff
@@ -114,7 +115,8 @@ namespace AZ
                     AZStd::vector<Semaphore::WaitSemaphore>(),
                     AZStd::vector<RHI::Ptr<Semaphore>>(),
                     {},
-                    &fence);
+                    &fence,
+                    nullptr); // TODO
```

Comment:
TODO

Reply:
Fixed this. This would have been a problem with more than one Fence in a Queue submit